libs/posthog_client_lite/src/lib.rs

```diff
@@ -547,10 +565,22 @@ impl PostHogClient {
     ) -> anyhow::Result<LocalEvaluationResponse> {
         // BASE_URL/api/projects/:project_id/feature_flags/local_evaluation
         // with bearer token of self.server_api_key
-        let url = format!(
-            "{}/api/projects/{}/feature_flags/local_evaluation",
-            self.config.private_api_url, self.config.project_id
-        );
+        // OR
+        // BASE_URL/api/feature_flag/local_evaluation/
+        // with bearer token of feature flag specific self.server_api_key
+        let url = if self.config.server_api_key.starts_with("phs_") {
+            // The new feature local evaluation secure API token
```

Comment:
You added `/api/feature_flag/local_evaluation` endpoint, but marked it as "old API with personal token"

The PR description says that you do the opposite: add a new secure API.

Shouldn't the "if" branches be swapped?

Reply:
yes you're right! fixed